src/apis/auth/index.ts

```
@@ -0,0 +1,21 @@
+import { instance } from '..'
```

ESLint

[error] 'instance' is defined but never used.

(@typescript-eslint/no-unused-vars)